src/pages/articles/cplusplus/Introduction to C++/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Introduction to C++
+---
+# Hello World! - Your first C++ Program 
+
+### What is C++ ?
+* C++ is a general purpose programming language which has been used since the 1990's
+* It is widely based on the programming language C and thus got it's name.
+
+### Here is your 1st program in C++:
+
+```C++
+#include <iostream>
+using namespace std ;
+int main()
+{
+    cout<<"Hello World"<<endl;
+    return 0;
+}
+```
+
```

Comment:
How about using a [repl.it](https://repl.it/) embed to use C++ in the browser with the above program and make this article more interactive?